### ISSUE TITLE
Add eslint for the front-end

### ISSUE THREAD

**Issue by Drarig29** [Drarig29/crypto-balance#3]
Currently there is no linting at all.

It would be cool if there was some.

**Comment by JohannesFischer** [comment]
I can help you out with that @Drarig29.

**Comment by Drarig29** [comment]
Nice! I can assign you the task if you want 😁

If possible, I'd like to enforce the use of semicolons and single quotes instead of double quotes.

**Comment by JohannesFischer** [comment]
Got it. Was thinking to base it of the `eslint:recommended` ruleset and make adjustments so it fits the existing coding style.

**Comment by Drarig29** [comment]
Yes, let's do this, but the existing coding style mixes single quotes and double quotes, so I'd like to enforce single ones 😉
